.github/workflows/daily.yml

```diff
@@ -0,0 +1,16 @@
+name: Scheduled build
+on:
+  branch_protection_rule:
+  schedule:
+    - cron: "12 9 * * *"
```

Comment:
the timestamp here is a little bit strange, you mean 9:12 UTC daily?

Reply:
When we run it daily, in fact any random time with in a day works, hence I use Sep 12th as the day I submit this PR. the Sep to hours and 12 to min.
I don't suppose it's harmful to run go sec and scorecard daily.
From security point of view, as new CVE been reported and announce randomly at any time. A daily check is better than a weekly with a higher frequency.